spec/libraries/azure_engine_spec.rb

```diff
@@ -73,9 +81,9 @@ def sign
   end
 end
 
-describe 'AzureEngine' do
+describe 'SlideHub::Cloud::Engine::Azure' do
   before do
-    AzureEngine.configure do |config|
```

Comment:
__[reek]__
* [UnusedParameters] has unused parameter 'container' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UnusedParameters] has unused parameter 'options' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)